src/Integration.UnitTests/LocalServices/ProjectSystemHelperTests.cs

```diff
@@ -53,13 +53,21 @@ public void TestInitialize()
         [TestMethod]
         public void ProjectSystemHelper_ArgCheck()
         {
-            Exceptions.Expect<ArgumentNullException>(() => new ProjectSystemHelper(null));
+            // Arrange
+            Action act = () => new ProjectSystemHelper(null);
+
+            // Act & Assert
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("serviceProvider");
```

Comment:
I added one new arg-check test to this class, so I changed the existing arg checks to use the Fluent library.